charts/generic/pod-delete/pod-delete.chartserviceversion.yaml

```diff
@@ -2,7 +2,7 @@ apiVersion: litmuchaos.io/v1alpha1
 kind: ChartServiceVersion
 metadata:
   name: pod-delete
-  version: 0.1.2
+  version: 0.1.1
```

Comment:
Leave the version as it is. We  should not be downgrading a version.

Reply:
fixed this